front/src/pages/Campaigns.vue

```diff
@@ -1,43 +1,57 @@
 <template>
   <q-page class="q-pa-md">
     <div class="column">
-      <div class="text-center">
-        <img style="width: 150px" src="../statics/weed.png" />
+      <div v-if="!items">
+        <NothingHere :text="text" :image="imgUrl"/>
```

Comment:
Please don't replace the entire page. We still want to show suggested campaigns below the component.